added support for xsiam (#80)
demisto/demisto-py: README.md

* DEMISTO_API_KEY
* DEMISTO_USERNAME
* DEMISTO_PASSWORD

> * DEMISTO_AUTH_ID (for XSIAM, If is set it means we should use the XSIAM api)

Reviewer: i think the env var should be named `XSIAM_AUTH_ID`

> * DEMISTO_AUTH_ID (for XSIAM, If is set it means we should use the XSIAM api)

Reviewer: ```suggestion
* DEMISTO_AUTH_ID (for Cortex XSIAM only. If set, Cortex XSIAM API will be used)
```

```

> **on the XSIAM we nead to add the auth_id**

Reviewer: ```suggestion
**For Cortex XSIAM, we need to set the auth_id**
```

```python
import demisto_client

> # Also possible to set env variables: DEMISTO_API_KEY and DEMISTO_BASE_URL

Reviewer: add note here about the possible env var
